crates/matrix-sdk-common/src/tracing_timer.rs

```diff
@@ -77,6 +77,12 @@ impl TracingTimer {
 ///
 /// The tracing level can be specified as a first argument, but it's optional.
 /// If it's missing, this will use the debug level.
+///
```

Comment:
Hmm, I would expect people to use the LSP to import things, the LSP knows how to import things for you.

Is this a useful comment? If so, why would we put it only on this macro?

Instead of this comment, perhaps it would make more sense to fix the first line in the docs and add an example:

> /// Macro to create a RAII timer that will log a `tracing` event once it's
/// dropped.

This doesn't explain what we're timing like the documentation for the struct does:

> /// A named RAII that will show on Drop how long its covered section took to
/// execute.

Adding an example to this will showcase how to use the macro as well as how to import it properly.

Reply:
> Hmm, I would expect people to use the LSP to import things, the LSP knows how to import things for you.

In this case, either the LSP didn't help me, or I was dumb and didn't trigger the LSP action to auto-import things :smiling_face_with_tear: 

Will add an example :+1: